modules/admission-webhook-deploy.adoc

```diff
@@ -0,0 +1,373 @@
+    name: server
+----
+<1> Delegates authentication and authorization to the webhook server API.
+<2> Allows the webhook server to access cluster resources.
+<3> Points to resources. This example points to the `namespacereservations` resource.
+<4> Enables the aggregated API server to create admission reviews.
+<5> Points to resources. This example points to the `namespacereservations` resource.
+<6> Enables the webhook server to access cluster resources.
+<7> Role binding to read the configuration for terminating authentication.
+<8> Default cluster role and cluster role bindings for an aggregated API server.
+
+. Apply those RBAC rules to the cluster:
++
+----
+$ oc auth reconcile -f rbac.yaml
+----
+
+. Create a YAML file called `webhook-daemonset.yaml` that is used to deploy a webhook as a DaemonSet server in a namespace:
++
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  namespace: my-webhook-namespace
+  name: server
+  labels:
+    server: "true"
+spec:
+  selector:
+    matchLabels:
+      server: "true"
+  template:
+    metadata:
+      name: server
+      labels:
+        server: "true"
+    spec:
+      serviceAccountName: server
+      containers:
+      - name: my-webhook-container  <1>
+        image: <image_registry_username>/<image_path>:<tag>  <2>
+        imagePullPolicy: IfNotPresent
+        command:
+        - <container_commands>  <3>
+        ports:
+        - containerPort: 8443 <4>
+        volumeMounts:
+        - mountPath: /var/serving-cert
+          name: serving-cert
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 8443 <5>
+            scheme: HTTPS
+      volumes:
+      - name: serving-cert
+        secret:
+          defaultMode: 420
+          secretName: server-serving-cert
+----
+<1> Note that the webhook server might expect a specific container name.
+<2> Points to a webhook server container image. Replace `<image_registry_username>/<image_path>:<tag>` with the appropriate value.
+<3> Specifies webhook container run commands. Replace `<container_commands>` with the appropriate value.
+<4> Defines the target port within pods. This example uses port 8443.
+<5> Specifies the port used by the readiness probe. This example uses port 8443.
+
+. Deploy the DaemonSet:
++
+----
+$ oc apply -f webhook-daemonset.yaml
+----
+
+. Define a secret for the service serving certificate signer, within a YAML file called `webhook-secret.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: my-webhook-namespace
+  name: server-serving-cert
+type: kubernetes.io/tls
+data:
+  tls.crt: <server_certificate>  <1>
+  tls.key: <server_key>  <2>
+----
+<1> References the signed webhook server certificate. Replace `<server_certificate>` with the appropriate certificate in base64 format.
+<2> References the signed webhook server key. Replace `<server_key>` with the appropriate key in base64 format.
+
+. Deploy the secret:
```

Comment:
Is "Deploy" the correct terminology? Should it just be "Create the secret" ?

Reply:
I agree, 'Create the secret' is better.